// Proto messages used by class_registration_test.cc.

syntax = "proto3";

import "google/protobuf/any.proto";

package tensorflow.serving;

option go_package = "github.com/tensorflow/serving/tensorflow_serving/util/class_registration_test_go_proto";

message Config1 {
  string string_field = 1;
}

message Config2 {
  string string_field = 1;
}

message MessageWithAny {
  google.protobuf.Any any_field = 1;
}
